client/src/store/cropper/actions.ts

```diff
@@ -0,0 +1,44 @@
+import {
+    SET_AVATAR_TO_CROPPER,
+    UPLOAD_AVATAR_PENDING,
+    UPLOAD_AVATAR_SUCCESS,
+    UPLOAD_AVATAR_ERROR,
+} from './actionTypes';
+import { Dispatch } from 'redux';
+import { AuthAPI } from '../api';
+import { showAlert } from '../alert/actions';
+
+export const setAvatarToCropper = (avatar: any): {type: string, payload: File} => ({
+    type: SET_AVATAR_TO_CROPPER,
+    payload: avatar,
+});
+
+export const uploadAvatarPending = (): { type: string } => ({
+    type: UPLOAD_AVATAR_PENDING,
+});
+
+export const uploadAvatarSuccess = (avatar: any): { type: string, payload: File } => ({
+    type: UPLOAD_AVATAR_SUCCESS,
+    payload: avatar,
+});
+
+export const uploadAvatarError = (error: Error): { type: string, payload: Error } => ({
+    type: UPLOAD_AVATAR_ERROR,
+    payload: error,
+});
+
+export const uploadAvatar = (avatar: File): (dispatch: Dispatch) => Promise<void> =>
+    async (dispatch: Dispatch): Promise<void> => {
+        try {
+            const data = new FormData();
+            data.append('userPhoto', avatar);
+            data.append('enctype', 'multipart/form-data');
```

Comment:
Pls move to reusable constant multipart...

Reply:
fixed